docs/rules/no-multi-assign.md

```diff
@@ -0,0 +1,44 @@
+# Disallow Use of Multiple Assignments Per Statement (no-multi-assign)
+
+Chaining the assignment of variables can lead to unexpected results or be difficult to read.
+
+```js
+a = b = c = d;
+```
+
+## Rule Details
+
+This rule disallows using multiple assignments within a single statement.
```

Comment:
I think this line might be a bit misleading. For example, this statement contains multiple assignments, but the rule allows it:

```js
foo = 1,
    bar = 2,
    baz = 3;
```

Maybe it would be more precise to say that the rule disallows chained assignment expressions.